hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/storage/BlockOutputStream.java

```diff
@@ -622,11 +613,9 @@ private void writeChunkToContainer(ChunkBuffer chunk) throws IOException {
       throw new IOException(
           "Unexpected Storage Container Exception: " + e.toString(), e);
     }
-    if (LOG.isDebugEnabled()) {
-      LOG.debug(
-          "writing chunk " + chunkInfo.getChunkName() + " blockID " + blockID
-              + " length " + effectiveChunkSize);
-    }
+    LOG.debug(
+        "writing chunk {} blockID {} length {}",
+        chunkInfo.getChunkName(), blockID, effectiveChunkSize);
```

Comment:
NIT: rewrite it as
      `LOG.debug("writing chunk {} blockID {} length {}",`
     `              chunkInfo.getChunkName(), blockID, effectiveChunkSize);`